devel/asio/Portfile

```diff
@@ -2,25 +2,34 @@
 
 PortSystem                  1.0
 PortGroup                   openssl 1.0
+PortGroup                   muniversal 1.0
+PortGroup                   github 1.0
 
-name                        asio
-version                     1.20.0
+github.setup                chriskohlhoff asio 1-22-1 asio-
+version                     1.22.1
```

Comment:
this should be done automatically by replacing the -" with "." using "${github.version}"